docs/source/config.json

```diff
@@ -39,6 +39,7 @@
       "Native Rust plugins": "/customizations/native",
       "Rhai scripts (experimental)": "/customizations/rhai"
     },
+    "Compatibility": "/compatibility",
```

Comment:
Except at the very top of the navigation, we try not to have single items that aren't categorized.

I recommend `Federation version support` instead of `Compatibility` and putting just below `"Migrating from @apollo/gateway"` above